packages/material-ui/src/InputLabel/InputLabel.js

```diff
@@ -141,7 +142,7 @@ InputLabel.propTypes = {
   children: PropTypes.node,
   /**
    * Override or extend the styles applied to the component.
-   * See [CSS API](#css-api) below for more details.
+   * See [CSS API](#css) below for more details.
```

Comment:
Nice finding!

Reply:
We should fix all the other links (+100)

Reply:
@oliviertassinari thanks for reviewing. Should I make separate PR for this?

Reply:
Yes, it would be awesome!